packages/website/package.json

```diff
@@ -14,7 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids",
     "typecheck": "tsc",
     "changelog": "conventional-changelog -i docs/changelog.md -s -r 0 -c conventional_commit_config.js && echo \"# Change log\n\n$(cat docs/changelog.md)\" > docs/changelog.md",
-    "contributors": "node contributors.js"
+    "contributors": "node contributors.js",
+    "publish:site": "npm run changelog && npm run contributors && npm run build"
```

Comment:
We can replace this script by using `npm hooks`: 
- `pre-`
- `post-`

This is an example (not tested with real scripts):

```json
{
  "changelog": "bla bla bla",
  "contributors": "bam bam bam",
  "build": "do amazing things",
  "prebuild": "npm changelog && npm contributors"
}
```

The `pre` hook would be run before the `build` script, keeping the `npm` (and other package managers) standards.

Reply:
Done

Reply:
I see you removed the script but I see no pre- hook in the package json

Reply:
@Cadienvan focused on the content and not the command, my bad, check it now;  "npm changelog" didn't worked, added run.